### PR TITLE
docs: add 004 CN version; modify 004.md and README

### PR DESCRIPTION
**Pull request by JoannaWu84**

### CODE REVIEW

File: newsletters/004.cn.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
++ [【Milk-V Duo 开发板免费体验】在 Windows 环境下点亮 Milk-V Duo](https://bbs.elecfans.com/jishu_2369979_1_1.html)
+
++ [【Milk-V Duo 开发板免费体验】Ubuntu 下烧录 linux 系统到 sd 卡，并在开发板上运行 linux 系统](https://bbs.elecfans.com/jishu_2369644_1_1.html)
+
++ [【Milk-V Duo 开发板免费体验】在 Archlinux 使用 schroot 建立 Milk-V Duo 开发环境](https://forum.sophgo.com/t/milk-v-duo-archlinux-schroot-milkv-duo/163)
+
++ [【Milk-V Duo 开发板免费体验】 Milk-V Duo 运行 openEuler RISC-V](https://forum.sophgo.com/t/milk-v-duo-milkv-duo-openeuler-risc-v/164)
+
++ [Milk-V Duo 试用报告在持续更新中，汇总链接如下](https://bbs.elecfans.com/try_CV1800B.html)
```
Comment (XieJiSS) [inline newsletters/004.cn.md:66]: 汇总链接指路

File: newsletters/004.md [hunk 2]
```diff
@@ -13,57 +19,53 @@ Most of the code is already open-source and can be obtained from repositories su
...
++ [Milk-V Duo Free Trial - Flashing system image in an Ubuntu environment and running Linux on Milk-V Duo](https://bbs.elecfans.com/jishu_2369644_1_1.html)
 
-  https://forum.sophgo.com/t/milk-v-duo-archlinux-schroot-milkv-duo/163
++ [Milk-V Duo Free Trial - Setting up the Milk-V Duo development environment with schroot in Archlinux](https://forum.sophgo.com/t/milk-v-duo-archlinux-schroot-milkv-duo/163)
 
-+ Milk-V Duo Free Trial - Running openEuler RISC-V in Milk-V Duo
++ [Milk-V Duo Free Trial - Running openEuler RISC-V in Milk-V Duo](https://forum.sophgo.com/t/milk-v-duo-milkv-duo-openeuler-risc-v/164)
 
-  https://forum.sophgo.com/t/milk-v-duo-milkv-duo-openeuler-risc-v/164
++ [Milk-V Duo's trial report continues to deliver. The summary link is below](https://bbs.elecfans.com/try_CV1800B.html)
```
Comment (XieJiSS) [inline newsletters/004.md:66]: Click to see summary links

**Review by XieJiSS — CHANGES_REQUESTED** [review]

File: newsletters/004.cn.md (hunk 1 above)
Comment (JoannaWu84) [reply newsletters/004.cn.md]: updated

**Review by XieJiSS — APPROVED** [review]
LGTM

Thanks!

**Pull request merged**